### DISCUSSION TITLE
v1.4.8-PRE

### DISCUSSION THREAD

**Discussion by MrMeowzz** [MrMeowzz/Funkin-MrMeowzz#38] [Prereleases]
- enemy now doesn't press poison, fire, and halo notes
- added b side in freeplay (will be added to story mode soon)
- pressing 5 now toggles notes and ratings and pressing 6 toggles rest of hud
- added missing discord rpc icons

<hr /><em>This discussion was created from the release <a href='https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/tag/v1.4.8-PRE'>v1.4.8-PRE</a>.</em>

**Comment by DeceptedPar** [comment]
thank you!
